i18n/en/design-system.json

```diff
@@ -43,5 +43,25 @@
   "global-footer-licensing-and-vertical-description-param-vertical-music": "Music",
   "global-footer-licensing-and-vertical-description-param-vertical-movies": "Movies",
   "global-footer-full-site-link": "View Full Site",
-  "global-footer-mobile-site-link": "View Mobile Site"
+  "global-footer-mobile-site-link": "View Mobile Site",
+  "global-navigation-fandom-logo-title": "Fandom",
+  "global-navigation-fandom-overview-community-central": "Community Central",
+  "global-navigation-fandom-overview-explore-wikis": "Explore Wikis",
+  "global-navigation-fandom-overview-link-vertical-games": "Games",
```

Comment:
agree!

how would you name the key?
vertical-games
vertical-tv
?

and we'll have to update footer...